gems/quilt_rails/lib/quilt_rails/configuration.rb

```diff
@@ -4,8 +4,10 @@ class Configuration
     attr_accessor :react_server_host, :react_server_protocol
 
     def initialize
-      @react_server_host = ENV['SERVICE_URL'] || 'localhost:8081'
-      @react_server_protocol = ENV['SERVICE_PROTOCOL'] || 'http'
+      ip = ENV['REACT_SERVER_IP']
```

Comment:
Nit: can we do the defaulting during assignment rather than string creation, please?  I find it easier to read interpolation when it's just var names.